src/test/java/org/opentestsystem/ap/imrt/common/model/ImrtItemTest.java

```diff
@@ -1,16 +0,0 @@
-package org.opentestsystem.ap.imrt.common.model;
```

Comment:
This was a temporary unit test to make sure the clover build task worked.  I normally don't test POJOs but can if we think there is value.